doc/infos/initial_position.md

```diff
@@ -0,0 +1,96 @@
+# At what position does the RxPlayer start a content ###########################
+
+## Overview ####################################################################
+
+When you give it a content to load, the RxPlayer has to set at one point the
+starting playback position.
+
+This documentation page explain how that position is calculated.
+
+Basically, we can separate four specific cases:
+
+  - a valid `startAt` option has been set to `loadVideo`, in which case we use
+    it to define the initial position.
+
+  - no `startAt` option has been set and we're playing a VoD content.
+
+  - no `startAt` option has been set and we're playing a live content.
+
+  - no `startAt` option has been set and we're playing a directfile content.
+
+
+## About the minimum and maximum position ######################################
+
+Regardless of the method, the minimum and maximum position of the content might
+be defined and used to calculate that starting position.
+
+Those position are inferred directly from the Manifest (when not playing a
+directfile content). Most Manifests declare segments currently available, in
```

Comment:
"Most Manifests declare every segments currently available" is more understandable.

Also this is false now, as the live edge is considered if given even in those cases